cluster-operator/src/main/java/io/strimzi/operator/cluster/model/AbstractModel.java

```diff
@@ -490,26 +490,20 @@ public String parseLogging(Logging logging, ConfigMap externalCm) {
 
             return createLog4jProperties(newSettings);
         } else if (logging instanceof ExternalLogging) {
-            if (((ExternalLogging) logging).getValueFrom() == null) {
-                if (externalCm != null && externalCm.getData() != null && externalCm.getData().containsKey(getAncillaryConfigMapKeyLogConfig())) {
-                    return maybeAddMonitorIntervalToExternalLogging(externalCm.getData().get(getAncillaryConfigMapKeyLogConfig()));
-                } else {
-                    log.warn("ConfigMap {} with external logging configuration does not exist or doesn't contain the configuration under the {} key. Default logging settings are used.",
-                            ((ExternalLogging) getLogging()).getName(),
-                            getAncillaryConfigMapKeyLogConfig());
-                    return createLog4jProperties(getDefaultLogConfig());
-                }
-            } else {
+            if (((ExternalLogging) logging).getValueFrom() != null && ((ExternalLogging) logging).getValueFrom().getConfigMapKeyRef() != null && ((ExternalLogging) logging).getValueFrom().getConfigMapKeyRef().getKey() != null) {
```

Comment:
We seem to do a lot of `((ExternalLogging) logging)` ... can we do it once in the beginning and use it in the whole block? It would clean the code nicely.